src/lamps/centurylamp.py

```diff
@@ -0,0 +1,51 @@
+from lamp import Lamp
+from behaviour import Behaviour
+from motion_6050 import LampMotionMPU6050
+import uasyncio as asyncio
+import random
+
+config = {
+    "base": { "pin": 12, "pixels": 40 },
+    "shade": { "pin": 13, "pixels": 40 },
+    "motion": { "pin_sda": 21, "pin_scl": 22, "polling_interval": 10e-2, "dance_gesture_peak": 23000 }
+}
+
+motion = LampMotionMPU6050(config["motion"]["pin_sda"], config["motion"]["pin_scl"])
```

Comment:
This could be handy to abstract so it's accessed simply via `lamp.motion` , and maybe have a `lamp.enable_motion()` or something that actually instantiates/enables it (simiiarly moving touch so it's not always setup)

then the enable could b e called within the run of the behaviour that uses it (enable would need to ensure it's not already enabled before doing anything, lest we do it multiple times)